# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/tests/non262/extensions/regress-466905-05.js
# Commit: 11fc8f557a2a
# Full Hash: 11fc8f557a2a7cf3c89e7c1396316aabd9725778
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2021-03-06 09:33:16
# Regressor Bug: 1689413
# File Overlap Count: 1
# Description:
#   Bug 1689413 part 6 - Move Realm* from ObjectGroup to BaseShape. r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106976
# ==============================================================================

diff -r 9bfa6023a553 -r 11fc8f557a2a js/src/tests/non262/extensions/regress-466905-05.js
--- a/js/src/tests/non262/extensions/regress-466905-05.js	Sat Mar 06 01:05:20 2021 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,42 +0,0 @@
-// |reftest| skip-if(!xulRuntime.shell) -- requires evalcx
-/* -*- Mode: C++; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-//-----------------------------------------------------------------------------
-var BUGNUMBER = 466905;
-var summary = 'Sandbox shapes';
-var actual = '';
-var expect = '';
-
-
-//-----------------------------------------------------------------------------
-test();
-//-----------------------------------------------------------------------------
-
-function test()
-{
-  printBugNumber(BUGNUMBER);
-  printStatus (summary);
-
-  if (typeof evalcx != 'function')
-  {
-    expect = actual = 'Test skipped: requires evalcx support';
-  }
-  else if (typeof shapeOf != 'function')
-  {
-    expect = actual = 'Test skipped: requires shapeOf support';
-  }
-  else
-  {
-
-    var s1 = evalcx('lazy');
-    var s2 = evalcx('lazy');
-
-    expect = shapeOf(s1);
-    actual = shapeOf(s2);
-  }
-
-  reportCompare(expect, actual, summary);
-}